python/python/lancedb/table.py

```diff
@@ -1784,9 +1793,23 @@ def _sanitize_nans(data, fill_value, on_bad_vectors, vec_arr, vector_column_name
     return data
 
 
-class AsyncTable(ABC):
+class AsyncTable(object):
```

Comment:
If it's just object, I don't think the parent class is necessary.
```suggestion
class AsyncTable:
```